### PR TITLE
Debian/buster

### PR DESCRIPTION
**Pull request by XECDesign**

### CODE REVIEW

File: rpi-eeprom-update [hunk 1]
```diff
@@ -370,15 +394,27 @@ those devices to stop working until after the system is reboot.
...
+
 Before selecting a firmware release directory this script checks whether there
 is a board revision specific variant e.g. critical-c03111. If present then the
 board-revision specific version is used in preference.
 
+Release status:
+
+critical: The latest production release plus important security or hardware compatibility bug fixes.
+
+stable: Contains new features have already undergone some beta testing.
```
Comment (lurch) [inline rpi-eeprom-update:407]: "features have" -> "features that have"

File: rpi-eeprom-update [hunk 2]
```diff
@@ -370,15 +394,27 @@ those devices to stop working until after the system is reboot.
...
+These are candidats for new production releases.
```
Comment (lurch) [inline rpi-eeprom-update:408]: candidats -> candidates

Comment (timg236) [reply rpi-eeprom-update:408]: That would need to be resolved via a PR to the script

Comment (lurch) [reply rpi-eeprom-update:408]: :+1:  #138

**Pull request merged**